### PR TITLE
chore: clean up SVGs to prevent clashes

### PR DESCRIPTION
**Pull request by Highjhacker**
## Summary

https://app.clickup.com/t/1p0dhxc

## Checklist

-   [x] I checked my UI changes against the design and there are no notable differences
-   [x] I checked my UI changes for any responsiveness issues
-   [x] I checked my UI changes in light AND dark mode
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

File: resources/icons/monitor.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><style>.monitor-st0{fill:none;stroke:currentColor;stroke-width:2;stroke-linecap:round;stroke-linejoin:round}</style><g id="Market_Cap_Fees_Monitor" transform="translate(-57.246 -1397.998)"><g id="Group_169" transform="translate(349.498 1690.998)"><g id="analytics-pie-1"><path id="Shape_1038" class="monitor-st0" d="M-280.1-292v6.8h6.8c.1-3.8-3-6.8-6.8-6.8z"/><path id="Shape_1039" class="monitor-st0" d="M-283.5-289.4c-4.2 0-7.7 3.4-7.7 7.7 0 4.2 3.4 7.7 7.7 7.7 4.2 0 7.7-3.4 7.7-7.7h-7.7v-7.7z"/></g></g></g></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><style>.monitor-st0{fill:none;stroke:currentColor;stroke-width:2;stroke-linecap:round;stroke-linejoin:round}</style><path class="monitor-st0" d="M12.152 1v6.8h6.8c.1-3.8-3-6.8-6.8-6.8z"/><path class="monitor-st0" d="M8.752 3.6c-4.2 0-7.7 3.4-7.7 7.7 0 4.2 3.4 7.7 7.7 7.7 4.2 0 7.7-3.4 7.7-7.7h-7.7V3.6z"/></svg>
```
Comment (leMaur) [inline resources/icons/monitor.svg]: `.monitor-st0` 👀

File: resources/icons/price.svg [hunk 2]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><style>.price-st0{fill:none;stroke:currentColor;stroke-width:2;stroke-linecap:round;stroke-linejoin:round}</style><path class="price-st0" d="M3.675 11.668c.071-.071.212-.071.283 0s.07.212 0 .283-.212.07-.283 0c-.141 0-.07-.213 0-.283m8.556-3.606c.071-.071.212-.071.283 0s.07.212 0 .282-.212.071-.283 0-.07-.212 0-.282"/><path class="price-st0" d="M1.271 12.516a1.002 1.002 0 010-1.414l7.92-7.92a1.002 1.002 0 011.414 0l4.313 4.314a1 1 0 010 1.414l-7.92 7.92a1.002 1.002 0 01-1.414 0l-4.313-4.314z"/><path class="price-st0" d="M9.332 11.243c.707-.707.636-1.767 0-2.545-.707-.707-1.768-.636-2.545 0-.637.778-.566 1.838 0 2.545.777.637 1.838.708 2.545 0zm5.091-8.061l4.314 4.314c.424.424.353 1.06 0 1.414l-7.99 7.99"/></svg>
```
Comment (leMaur) [inline resources/icons/price.svg]: `.price-st0` 👀

File: resources/icons/transaction.svg [hunk 3]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><style>.transaction-st0,.transaction-st1{fill:none;stroke:currentColor;stroke-width:2;stroke-linecap:round}.transaction-st1{stroke-miterlimit:4.0005}</style><g id="Transaction_Type" transform="translate(24.5 -1294.5)"><g id="Сгруппировать_775"><path id="Линия_24" class="transaction-st0" d="M-17.8 1297.4v14.2"/><path id="Линия_26" class="transaction-st1" d="M-12.1 1303.1l6.6 8.2"/><path id="Линия_27" class="transaction-st1" d="M-5.5 1303.1l-6.6 8.2"/></g><path id="Линия_25" class="transaction-st0" d="M-23.5 1297.4h11.4"/></g></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><style>.transaction-st0{fill:none;stroke:currentColor;stroke-width:2;stroke-linecap:round}</style><path class="transaction-st0" d="M6.7 2.9v14.2"/><path d="M12.4 8.6l6.6 8.2m0-8.2l-6.6 8.2" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"/><path class="transaction-st0" d="M1 2.9h11.4"/></svg>
```
Comment (leMaur) [inline resources/icons/transaction.svg]: `.transaction-st0` 👀

File: resources/icons/transactions.svg [hunk 4]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><style>.transactions-st0{fill:none;stroke:currentColor;stroke-width:2;stroke-linecap:round;stroke-linejoin:round}</style><g id="Transactions" transform="translate(-322.003 -6006)"><g id="Group_605"><g id="synchronize-arrows-square-2"><path id="Shape_2583" class="transactions-st0" d="M326 6014.8v5.4c0 1.3 1 2.4 2.4 2.4h7.1c1.3 0 2.4-1 2.4-2.4v.4"/><path id="Shape_2584" class="transactions-st0" d="M337.9 6017.1v-5.3c0-1.3-1-2.4-2.4-2.4h-7.1c-1.3 0-2.4 1-2.4 2.4v-.4"/><path id="Shape_2585" class="transactions-st0" d="M329 6017.7l-3-3-3 3"/><path id="Shape_2586" class="transactions-st0" d="M335 6014.2l3 3 3-3"/></g></g></g></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><style>.transactions-st0{fill:none;stroke:currentColor;stroke-width:2;stroke-linecap:round;stroke-linejoin:round}</style><path class="transactions-st0" d="M3.997 8.8v5.4c0 1.3 1 2.4 2.4 2.4h7.1c1.3 0 2.4-1 2.4-2.4v.4m0-3.5V5.8c0-1.3-1-2.4-2.4-2.4h-7.1c-1.3 0-2.4 1-2.4 2.4v-.4"/><path class="transactions-st0" d="M6.997 11.7l-3-3-3 3m12-3.5l3 3 3-3"/></svg>
```
Comment (leMaur) [inline resources/icons/transactions.svg]: `.transaction-st0` 👀

**Review by leMaur — CHANGES_REQUESTED** [review]

**Comment by leMaur** [comment]
@Highjhacker if you have checked that the design of each icon is unchanged... then the optimizations are ok for me! 🚀

**Review by ItsANameToo — APPROVED** [review]

**Comment by ItsANameToo** [comment]
@Highjhacker conflict

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**